src/core/1/plugins/plugins/events/wildcard-events/index.md

```diff
@@ -0,0 +1,86 @@
+---
+type: page
+code: false
+title: Wildcard Events
+order: 250
+---
+
+# Wildcard Events
+
+Kuzzle allows you to listen to multiple events with a single listener thanks to wildcards (`*`).
+
+---
+
+## Single wildcard
+
+| Arguments | Type                                                           | Description                |
+| --------- | -------------------------------------------------------------- | -------------------------- |
+| `request` | <pre><a href=/plugins/1/constructors/request>Request</a></pre> | The normalized API request |
+
+With a single wildcard you can listen to all the events triggered by Kuzzle.
+
+#### Example
+
+```javascript
+this.pipes = 
+  // listener function will be triggered at each event
+  '*': <function to call> 
+};
+```
+
+---
+
+## Name wildcard
+
+| Arguments | Type                                                           | Description                |
+| --------- | -------------------------------------------------------------- | -------------------------- |
+| `request` | <pre><a href=/plugins/1/constructors/request>Request</a></pre> | The normalized API request |
+
+You can also use wildcards to replace only the name of an event.
+
+### Naming Template
+
+The event name is built using the following template:
```

Comment:
There are other events than API events. Check the documentation: https://docs.kuzzle.io/plugins/1/events/intro/